services/agent-router/internal/httpapi/route_test.go

```
@@ -0,0 +1,582 @@
+// POST /v1/route (STORY-035-10 gate 9). Subtest names are the literal
+// tokens gate 9 greps for; do not rename them without updating the gate.
+func TestRouteDecisionTable(t *testing.T) {
+	schema := testutil.CompileSchema(t, "execution-profile.schema.json")
+	errSchema := testutil.CompileSchema(t, "error.schema.json")
+
+	// contained: ordinary contained code change -> local-code-standard,
+	// vendor fallbacks in committed priority order
+	// (examples/execution-profile/local-code-standard.json).
+	t.Run("contained", func(t *testing.T) {
+		e := newEnv(t, realCatalogState(t))
+		resp := postRoute(t, e, callerToken, routeRequestBody(t, contained200()))
```

golangci-lint (2.12.2)

[error] response body must be closed

(bodyclose)